code/modules/clothing/outfits/standard.dm

```diff
@@ -369,7 +369,7 @@
 /datum/outfit/death_commando
 	name = "Death Commando"
 
-	uniform = /obj/item/clothing/under/color/green
+	uniform = /obj/item/clothing/under/syndicate/combat
```

Comment:
Why?

Reply:
because why not, it would only make sense for an combat strike team to actually war *combat* unfiroms instead of green jumpsuits, if you dont like this i can change it to centcom jumpsuits.